examples/ui5-js-app/webapp/Component.js

```diff
@@ -27,6 +27,20 @@ sap.ui.define(
 
                 // set the device model
                 this.setModel(models.createDeviceModel(), "device")
+
+                const url = new URL(location.href)
+                if (url.searchParams.get("isui5toolingTest")?.toLocaleLowerCase() === "true") {
+                    const startXHR = () => {
+                        this.getModel().read("/Categories", {
+                            success: startXHR
+                        })
+                    }
+                    startXHR()
+                    const startFetch = () => {
```

Comment:
i keep my fingers crossed that the gh runners are equipped with enough memory for the CI runs 😁